types/react-cytoscapejs/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for react-cytoscapejs 1.1
+// Project: https://github.com/plotly/react-cytoscapejs
+// Definitions by:  Emmanuel COunasse <https://github.com/manuc66/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import cytoscape = require('cytoscape');
+import { Component, CSSProperties } from 'react';
+
+export interface CytoscapeComponentProps {
+    id?: string;
+    cy?: (cy: cytoscape.Core) => void;
+    style?: CSSProperties;
+    elements: cytoscape.ElementDefinition[];
+    layout?: cytoscape.LayoutOptions;
+    stylesheet?: cytoscape.Stylesheet | cytoscape.Stylesheet[] | string;
+    className?: string;
+    zoom?: number;
+    pan?: Position;
+    minZoom?: number;
+    maxZoom?: number;
+    zoomingEnabled?: boolean;
+    userZoomingEnabled?: boolean;
+    boxSelectionEnabled?: boolean;
+    autoungrabify?: boolean;
+    autounselectify?: boolean;
+}
+
+export default class CytoscapeComponent extends Component<CytoscapeComponentProps> {
```

Comment:
```suggestion
export = CytoscapeComponent;
class CytoscapeComponent extends Component<CytoscapeComponentProps> {
```

_____

This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Variables

```ts
declare var xyz: SomeType;

export = xyz;
```

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

To import the entity using a default import, you will need to enable `esModuleInterop`. Alternatively, you can just import using a more appropriate syntax like

```ts
import foo = require("foo");
```

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).